### DISCUSSION TITLE
autofix?

### DISCUSSION THREAD

**Discussion by LLNobles** [community/community#141385] [New to GitHub]
### Select Topic Area

Product Feedback

### Body

If I am discussing the implementation of auto-fix in the technical world, it may be helpful to some users. However, I have yet to fully develop my coding abilities. I am new to the community.

**Comment by ghostinhershell** [comment]
Thanks for posting in the GitHub Community, @LLNobles!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. This question should be in the New to GitHub category. The New to GitHub category is solely related to conversations around getting started on using GitHub and the community for beginners of GitHub, Community on Discussions, and those new to different areas of the product alike! I've gone ahead and moved it for you. Good luck!
